FRW/Services/Network/FlowNetwork.swift

```diff
@@ -1164,6 +1164,18 @@ extension FlowNetwork {
         ).decode(Flow.StorageInfo.self)
         return response
     }
+    
+    static func checkAccountInfo() async throws -> Flow.AccountInfo {
+        guard let address = WalletManager.shared.getPrimaryWalletAddress().map(Flow.Address.init(hex:)) else {
+            throw LLError.invalidAddress
+        }
+                                                           
+        guard let cadence = CadenceManager.shared.current.basic?.getAccountInfo?.toFunc() else {
+            throw LLError.invalidCadence
+        }
+        
+        return try await flow.accessAPI.executeScriptAtLatestBlock(cadence: cadence, arguments: [.address(address)]).decode(Flow.AccountInfo.self)
```

Comment:
We should use one func to make all query, which allow us tracking and report to Mixpanel. 
@zhouxl  Have we merge the change for KeyPath method for mixpanel ?